media/css/firefox/new/fx-lifestyle-base.less

```diff
@@ -0,0 +1,283 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../sandstone/lib.less';
+
+@blue: #0a99d5;
+@grey: #EFEFEF;
+
+// generates a < tablet media query for all sections (except intro)
+.common-mobile-props(@padding: 250px) {
+    @media only screen and (max-width: @breakTablet) {
+        background-position: bottom 20px center;
+        min-height: 0;
+        padding-bottom: @padding;
+
+        .copy {
+            float: none;
+            width: auto;
+        }
+    }
+}
+
+#outer-wrapper {
+    background: @grey;
+}
+
+#wrapper {
+    background: #fff;
+    width: auto;
+
+    // add icon to download buttons *not* in family nav
+    .download-button .download-title:after {
+        background: url('/media/img/firefox/family/index-b/download-arrow.svg') center right no-repeat;
+        content: '';
+        display: inline-block;
+        height: 20px;
+        margin: 0 0 -3px 12px;
+        width: 20px;
+    }
+}
+
+#intro {
+    &.content-section {
+        padding-top: 60px;
+    }
+
+    .copy {
+        float: right;
+    }
+
+     @media only screen and (max-width: @breakTablet) {
+        .copy {
+            padding-right: 0;
+        }
+    }
+
+    .content {
+        background-position: bottom center;
+        text-align: right;
+        color: white;
+    }
+
+    h2 {
+        color: white;
+    }
+
+    p {
+        .font-size(24px);
+        line-height: 38px;
+        margin-bottom: 50px;
+    }
+}
+
+#tabzilla:before {
+    background-color: @blue;
+}
+
+#newsletter-form {
+    display: none;
+}
```

Comment:
Instead of doing this, it would be better to just override the block in the template. Adding `{% block email_form %}{% endblock %}` to `fx-lifestyle-base.html` would be cleaner.